dark-mode.css

```
@@ -681,3 +681,6 @@ body.dark-mode .rcx-box--text-color-default,
 body.dark-mode .rcx-box--text-color-info {
     color: var(--color-dark);
 }
+body.dark-mode .rcx-\@tmocsy { 
+	background-color: #1d74f5 !important;
+}
```

Unfortunately `.rcx-\@tmocsy` is a generated class from React. I don't think depending on this class name is a good idea since it could change from instance to instance or version to version. But we can target the mark via it's aria-label instead:
```suggestion
body.dark-mode div.rcx-box.rcx-box--full[aria-label="Unread"] { 
	background-color: #1d74f5 !important;
}
```

---

@ankar84 Let me know what you think of the above suggestion.

---

@pbaity definitely you a right!
I thought that label it's not a best to use. So label would be great!

---

You're right, I think using labels is usually not recommended, but if we have to choose between styling via label or via a generated class name, I _think_ labels would be better. I'm not a CSS expert by any means so anyone can feel free to suggest otherwise.